src/zabapgit_popups.prog.abap

```diff
@@ -341,7 +341,7 @@ CLASS lcl_popups IMPLEMENTATION.
       READ TABLE lt_fields WITH KEY fieldname = 'LINE'
                            ASSIGNING <ls_field>.
       ASSERT sy-subrc = 0.
-      ev_name = lcl_tag=>add_tag_prefix( <ls_field>-value ).
```

Comment:
missing deletion of the lcl_tag class? just remove the code, not the include yet, we'll delete the empty includes later to make it easier for end users